PTFE-1366 - Migrate registry containers for spark to ghcr.io (#52)
scality/spark: ansible/Readme.md

* Use an SSH Agent with the SSH key added that can connect to the inventory host
in the `[staging]` group

> * Set the REGISTRY_USER variable to match a ghcr.io API user 

Reviewer: Honestly this can be anything now, all that matters for ghcr.io is the password. But saying API user might be a bit broad:

```suggestion
* Set the REGISTRY_USER variable to your GitHub username. 
```

> * Set the REGISTRY_PASSWORD variable to matching API password

Reviewer: ```suggestion
* Set the REGISTRY_PASSWORD variable to a [GitHub PAT](https://github.com/settings/tokens) with `packages:read` permissions.
```

  -e "REGISTRY_USER=User_Name" \

>   -e "REGISTRY_PASSWORD=<CLI/API_KEY>" \

Reviewer: ```suggestion
  -e "REGISTRY_PASSWORD=<GitHub PAT>" \
```
